src/main/java/school/faang/Group_Users_By_Age/User.java

```diff
@@ -0,0 +1,36 @@
+package school.faang.Group_Users_By_Age;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class User {
+    private String name;
+    private int age;
+    private String placeWork;
+    private String address;
+
+    public static Map<Integer, List<User>> groupUsers(List<User> userList){
```

Comment:
Засчитано, но можно короче, у Map есть удобные для этого методы:

```suggestion
    public static Map<Integer, List<User>> groupUsers(List<User> userList) {
    Map<Integer, List<User>> map = new HashMap<>();

    for (User user : userList) {
        map.computeIfAbsent(user.getAge(), k -> new ArrayList<>()).add(user);
    }

    return map;
}
```